src/app/learn/reference/configuration/index.html

```diff
@@ -14,7 +14,7 @@
     <stache-code>$schema</stache-code>
   </stache-page-anchor>
   <p>
-    Specifies which file is used to provide annotations and validation to the entire configuration file in your IDE. This file should not be modified.
+    The <stache-code>$schema</stache-code> configuration option specifies the file that provides annotations and validation to the entire configuration file in your IDE. You should not modify this file should.
```

Comment:
Extra should at the end of the second sentence.

Reply:
Nice catch. Thanks!